components/setting_item_min.jsx

```diff
@@ -80,6 +80,7 @@ export default class SettingItemMin extends React.PureComponent {
                         className='color--link cursor--pointer style--none'
                         onClick={this.handleUpdateSection}
                         ref={this.getEdit}
+                        aria-labelledby={this.props.section + 'Title ' + this.props.section + 'Edit'}
```

Comment:
I believe @asaadmahmood wanted these to be lower case, maybe he can confirm

Reply:
Yup, we can just use `toLowerCase()` function at the end of any labels we are reusing.
Also, these should be translatable.
So I'm sure we have the Edit text used in a translatable state in some of the components which you can have a look at.
Here is one example.
```const ariaLabel = `${props.accessibilityLabel} ${formatMessage({id: 'accessibility.button.dialog', defaultMessage: 'Dialog'})}`;```

Reply:
I see that you're using something similar to the `id` that is given. But for aria-labels, we usually want to translate them.

Reply:
So by using `aria-labelledby`, you end up reading the text that should already be translated elsewhere. Unless I'm missing something, or is this a best practices issue?

Reply:
Yup, you are right. So it's basically reading the content, which is fine, and in this case, I think it's fine not to have them lowercase, as it is linking to the content of an element rather than an aria-label directly.